pkgs/development/python-modules/ibis-framework/default.nix

```
@@ -17,23 +17,25 @@
 , geopandas
 , google-cloud-bigquery
 , google-cloud-bigquery-storage
-, graphviz-nox
+, graphviz
```

```suggestion
, graphviz
```
why is this changed?

---

Yeah, ideally this can be left as `graphviz-nox`. Ibis only uses graphviz to construct `dot` syntax, so no need to bring in X11

---

This is `python3Packages.graphviz`. We can make that package use `graphviz-nox` if you want.